developers/weaviate/api/graphql/additional-operators.md

```diff
@@ -463,14 +494,18 @@ To sort by more than one property, pass an array of { `path`, `order` } objects
 </Tabs>
 
 
-#### Additional properties
+#### Metadata properties
 
-Sometimes sorting by an additional property is required, such as `id`, `creationTimeUnix`, or `lastUpdateTimeUnix`.  This can be achieved by prefixing the property name with an underscore.
+To sort with metadata, add an underscore to the property name.
 
-For example:
+| Property Name | Python Client 4 Name | Sort Property  Name |
```

Comment:
I find this a little confusing to be honest as the second column is irrelevant to sorting operations. Can we drop the second column please?